From e4f1e956cf8f7d1ee81213a42396412bfad36588 Mon Sep 17 00:00:00 2001
From: Ronald Claveau <linux-kernel-dev@aliel.fr>
Date: Fri, 3 Apr 2026 12:08:16 +0200
Subject: [PATCH 75/91] dt-bindings: phy: amlogic: usb2: Add compatible for
 Amlogic T7 SoC

The Amlogic T7 USB2 PHY uses the same register layout
and initialization sequence as the G12A USB2 PHY.
Add amlogic,t7-usb2-phy with a fallback to amlogic,g12a-usb2-phy.

Signed-off-by: Ronald Claveau <linux-kernel-dev@aliel.fr>
---
 .../bindings/phy/amlogic,g12a-usb2-phy.yaml           | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/phy/amlogic,g12a-usb2-phy.yaml b/Documentation/devicetree/bindings/phy/amlogic,g12a-usb2-phy.yaml
index 1a0c436b87a0..f7cd9d16dd12 100644
--- a/Documentation/devicetree/bindings/phy/amlogic,g12a-usb2-phy.yaml
+++ b/Documentation/devicetree/bindings/phy/amlogic,g12a-usb2-phy.yaml
@@ -12,9 +12,14 @@ maintainers:
 
 properties:
   compatible:
-    enum:
-      - amlogic,g12a-usb2-phy
-      - amlogic,a1-usb2-phy
+    oneOf:
+      - items:
+          - enum:
+              - amlogic,t7-usb2-phy
+          - const: amlogic,g12a-usb2-phy
+      - enum:
+          - amlogic,g12a-usb2-phy
+          - amlogic,a1-usb2-phy
 
   reg:
     maxItems: 1
-- 
2.53.0

